docs/articles/other/Account-Management.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Account Management
-description: Everything you need to know about Having an Expensify Account Manager
+title: Your Expensify Account Manager
```

Comment:
Ah this is the title for SEO, we should update the name of the file to display the article name to `Your-Expensify-Account-Manager.md`. But now I'm thinking if we should use this also for the title on the article page 🤔